src/completion-provider.js

```diff
@@ -110,6 +116,28 @@ class CompletionProvider {
       (suggestions) => this.filterSuggestions(prefix, suggestions)
     );
   }
+
+  lookupWebpack(prefix) {
+    const projectPath = atom.project.getPaths()[0];
+    if (!projectPath) {
+      return Promise.resolve([]);
+    }
+
+    const webpackConfigFilename = atom.config.get('autocomplete-modules.webpackConfigFilename');
+    const webpackConfigPath = path.join(projectPath, webpackConfigFilename);
+    const vendors = atom.config.get('autocomplete-modules.vendors');
+
+    const webpackConfig = require(webpackConfigPath);
+    let moduleSearchPaths = get(webpackConfig, 'resolve.modulesDirectories', []);
```

Comment:
Oh wait, I get what you're saying now. It's the direct `require` of the webpack config. DOH! Yes, a try/catch would be the correct solution here. Thanks.